source/security.rst

```diff
@@ -44,15 +50,20 @@ If the checks above found keys that do not have a passphrase, then you should ad
 
 If you have more than one key to add a passphrase to, you can get them all with this snippet::
 
-    shopt -s extglob; for keyfile in ~/.ssh/id_!(*.sock|*.pub); do ssh-keygen -f ${keyfile} -p -o -a 100 ; fi ; done
+    shopt -s extglob; for keyfile in ~/.ssh/id_!(*.sock|*.pub); do \
+       ssh-keygen -f ${keyfile} -p -o -a 100 ; \
+    done
 
 You may use the same passphrase for all your SSH keys. If you do, then `ssh-add` will let you add _all_ of them to your
 SSH agent at once, which will make it much easier to use multiple keys.
 
+(``-o`` = use newer file format, ``-a 100`` = number of KDF rounds,
```

Comment:
If this is documenting the snippet above, it should be moved just below the snippet